docs/source/bibliography.bib

```diff
@@ -442,3 +442,23 @@ @article{OneStop
     journal={PsyArXiv preprint},
     year={2025}
 }
+
+@InProceedings{ETDD70,
+    author="Sedmidubsky, Jan
+    and Dostalova, Nicol
+    and Svaricek, Roman
+    and Culemann, Wolf",
+    editor="Ch{\'a}vez, Edgar
+    and Kimia, Benjamin
+    and Loko{\v{c}}, Jakub
+    and Patella, Marco
+    and Sedmidubsky, Jan",
```

Comment:
isn't this person here listed twice?

Reply:
yea seems like it. but as editor and author, I clarified